docs/docs/tutorials/scalability/newspaper.md

```diff
@@ -1,5 +1,7 @@
 # Newspaper website
 
+**TODO**: Do we want this use case here? Feels a bit misplaced.
```

Comment:
Agreed. We should take parts of this and put it under the QAE service, most other things are covered by our new use case documentation hopefully.